torch/csrc/api/src/utils.cpp

```diff
@@ -10,7 +11,7 @@ void manual_seed(uint64_t seed) {
   at::globalContext().defaultGenerator(at::kCPU).manualSeed(seed);
   // NB: Sometimes we build with CUDA, but we don't have any GPUs
   // available. In that case, we must not seed CUDA; it will fail!
-  if (at::globalContext().hasCUDA() && at::globalContext().getNumGPUs() > 0) {
+  if (cuda::is_available() > 0) {
```

Comment:
ok this is valid but comparing a bool > 0 is just confusing...

Reply:
No I just fucked this one up XD